syft/pkg/cataloger/package_url_test.go

```diff
@@ -24,9 +24,30 @@ func TestPackageURL(t *testing.T) {
 		},
 		{
 			pkg: pkg.Package{
-				Name:    "name",
-				Version: "v0.1.0",
+				Name:    "bad-name",
+				Version: "bad-v0.1.0",
 				Type:    pkg.PythonPkg,
+				Metadata: pkg.PythonPackageMetadata{
```

Comment:
Thanks for adding these cases! Since we're adding new cases to this `TestPackageURL` test it might be good to take the opportunity to add a `name` field to the test struct `lines 13-15`. This field should just be used as a label to state which case the test struct is covering so when we come back to look at this file we have context for the different tests under `TestPackageURL`

Reply:
Added - also backfilled it for some other tests. :)